tangelo/toolboxes/operators/operators.py

```diff
@@ -75,6 +77,10 @@ def __iadd__(self, other):
                 f_op.terms = other.terms.copy()
                 return super(FermionOperator, self).__iadd__(f_op)
 
+        elif isinstance(other, COEFFICIENT_TYPES):
```

Comment:
what motivated this change ? Were you getting errors when trying to add FemrionOperators with some types on this list because that addition was not supported by Openfermion ?

Reply:
Thank you very much